src/sql/platform/connection/common/connectionProfileGroup.ts

```diff
@@ -9,14 +9,18 @@ import { isUndefinedOrNull } from 'vs/base/common/types';
 import { assign } from 'vs/base/common/objects';
 import { find } from 'vs/base/common/arrays';
 
-export interface IConnectionProfileGroup {
-	id: string;
+export interface INewConnectionProfileGroup {
+	id?: string;
 	parentId?: string;
 	name: string;
 	color?: string;
 	description?: string;
 }
 
+export interface IConnectionProfileGroup extends INewConnectionProfileGroup {
+	id: string;
```

Comment:
Hmm I see that it's defined here...

Reply:
anything previously using IConnectionProfileGroup won't see a change in the definition. I basically just added a new interface where the id is optional.